verible/common/formatting/format-token.h

```diff
@@ -28,6 +28,13 @@
 
 namespace verible {
 
+// TODO: find something platform independent that is less ugly.
+// Or maybe revisit the place where we need this and see if they could
+// be actually represented by a const char * nullptr.
+inline absl::string_view::iterator string_view_null_iterator() {
```

Comment:
this should probably in a different file; or in no file at all, just using `absl::string_view::iterator{}` everywhere (but that would then be less explicit).

Reply:
maybe place in `common/strings`?

Reply:
For now, I narrowed its use to anything that anyway uses `format-token.h`, so decided to leave it there for now.
(otherwise it would be a bit awkward to have a `strings/null-iterator.h` with a single inlined one-liner function in it...)